src/BagnoDB.Serializator/OptionSerializer.fs

```diff
@@ -21,7 +21,8 @@ type OptionSerializer<'TOption when 'TOption: equality>() =
 
         let (case, args) =
             let value =
-                if (typeOfArg.IsPrimitive) then
+                if typeOfArg.IsPrimitive 
```

Comment:
Since we have `3 checks` here I would prefer to extract it as variable like:

```fsharp
let isPrimitiveOrDecimal = ...
if isPrimitiveOrDecimal then
    ...
...
```

Reply:
In latest commit